src/uk/gov/defra/ffc/DefraUtils.groovy

```diff
@@ -179,5 +179,27 @@ def triggerDeploy(jenkinsUrl, jobName, token, params) {
   sh(script: "curl -k $url")
 }
 
+def notifySlackBuildFailure(exception, channel) {
```

Comment:
There's a bit of duplication in the if/else statement.  The only changes are the channel name and the @here addition.  

Could we update the channel variable with the master build channel if the build is master.  Then we'd only need the slackSend command once.
```

Reply:
i have change the function to only call slackSend once, i think we could make it so that the msg is only defined once as well but wanted to see if this is what you meant first

Reply:
Yep, that's what I meant.